events/wc_titanforged_events.txt

```diff
@@ -1 +1,218 @@
+					character = event_target:target_thorim
+					event_target:target_loken = { ai = yes }
+				}
+			}
+		}
+		character_event = { id = WCUTF.2 days = 7 }
+		
+		ai_chance = {
+			factor = 0
+		}
+	}
+	option = {
+		name = EVTOPTC_WCUTF.1
+		trigger = {
+			OR = {
+				character = event_target:target_loken
+				AND = {
+					character = event_target:target_thorim
+					event_target:target_loken = { ai = yes }
+				}
+			}
+		}
+		character_event = { id = WCUTF.3 days = 7 }
+		
+		ai_chance = {
+			factor = 0
+		}
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.2
+	desc = {
+		trigger = {
+			character = event_target:target_loken
+		}
+		text = EVTDESC_WCUTF.2_loken
+	}
+
+	desc = {
+		trigger = {
+			character = event_target:target_thorim
+		}
+		text = EVTDESC_WCUTF.2_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.2
+		trigger = {
+			character = event_target:target_loken
+		}
+		religion = old_gods_worship
+	}
+	option = {
+		name = EVTOPTB_WCUTF.2
+		trigger = {
+			character = event_target:target_thorim
+		}
+		religion = old_gods_worship
+		event_target:target_loken = {
+			this_abdicate_transfer_wealth_to_prev_effect = yes
+			
+			death = { death_reason = death_duel killer = PREV }
+		}
+	}
+	after = {
+		d_ulduar = { any_de_jure_vassal_title = { location = { remove_province_modifier = ulduar_closed_down } } }
+		set_global_flag = ulduar_opened_flag
+	}
+}
+character_event = {
+	#Basic event information
+	id = WCUTF.3
+	desc = {
+		trigger = {
+			character = event_target:target_loken
+		}
+		text = EVTDESC_WCUTF.3_loken
+	}
+
+	desc = {
+		trigger = {
+			character = event_target:target_thorim
+		}
+		text = EVTDESC_WCUTF.3_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+
+	immediate = {
+	}
+	option = {
+		name = EVTOPTA_WCUTF.3
+		trigger = {
+			character = event_target:target_loken
+		}
+		event_target:target_thorim = {
+			event_target:target_loken = {
+				this_abdicate_transfer_wealth_to_play_prev_effect = yes
+				
+				death = { death_reason = death_suicide }
+			}
+		}
+	}
+	option = {
+		name = EVTOPTB_WCUTF.3
+		trigger = {
+			character = event_target:target_thorim
+		}
+		religion = mystery_of_the_makers
+		event_target:target_loken = {
+			this_abdicate_transfer_wealth_to_prev_effect = yes
+			
+			death = { death_reason = death_duel killer = PREV }
+		}
+	}
+	after = {
+		event_target:target_thorim = {
+			d_ulduar = {
+				any_de_jure_vassal_title = {
+					location = {
+						remove_province_modifier = ulduar_closed_down
+						religion = mystery_of_the_makers
+					}
+				}
+			}
+			any_realm_character = {
+				limit = { religion = cult_of_loken }
+				religion = mystery_of_the_makers
+			}
```

Comment:
Hide it or write tooltip plz. `WCUTF.2` has the same issue.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/78928610-f5365480-7ab1-11ea-855f-d06050f18594.png)

</details>